### PR TITLE
Adding regexp character classes

### PR DESCRIPTION
**Pull request by SphinxKnight**
As part of https://github.com/mdn/sprints/issues/1721

I was tempted to add an example with emojis as I was unable to do so, please let me know if this is not a good idea, even in interactive-examples :s

### CODE REVIEW

File: live-examples/js-examples/regexp/regexp-character-classes.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js">var chessStory = "He played the King in a8 and she moved her Queen in c2.";
```
Comment (wbamberg) [inline live-examples/js-examples/regexp/regexp-character-classes.html]: We're using `const` and `let` in new examples now, and slowly updating existing ones.

File: live-examples/js-examples/regexp/regexp-character-classes.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+var regexpCoordinates = /\w\d/g;
+console.log(chessStory.match(regexpCoordinates));
+// expected output [ 'a8', 'c2']
```
Comment (wbamberg) [inline live-examples/js-examples/regexp/regexp-character-classes.html]:
```suggestion
// expected output: Array ["a8", "c2"]
```

File: live-examples/js-examples/regexp/regexp-character-classes.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+var moods = "happy 🙂, confused 😕, sad 😢";
+var regexpEmoticons = /[\u{1F600}-\u{1F64F}]/gu;
+console.log(moods.match(regexpEmoticons));
+// expected output ['🙂', '😕', '😢']
```
Comment (wbamberg) [inline live-examples/js-examples/regexp/regexp-character-classes.html]:
```suggestion
// expected output: Array ["🙂", "😕", "😢"]
```

**Comment by SphinxKnight** [comment]
Thanks for the review Will :) (I'll be tackling the other PRs with var as well :))

**Pull request merged**